packages/buildTools/cli/cli.ts

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ts-node
+
+import { Command } from '@commander-js/extra-typings';
+
+import { version } from '../package.json';
+import { bindBuild } from './bindBuild';
+import { bindMigrate } from './bindMigrate';
+import { bindPrepareEnv } from './bindPrepareEnv';
+
+const program = new Command();
+
+program.name('@DDD forum build utils').description('CLI to build @DDD forum packages').version(version);
+
+bindBuild(program);
+bindPrepareEnv(program);
+bindMigrate(program);
```

Comment:
That's sweet. Very open-closed. We could add new scripts to this as we like. That's awesome. 

Is it right to call this a _build_ tool? Seems like we can use it for most of what we have to do.

Here's my current list of scripts/capabilities we will need from what I gather.

<img width="734" alt="Screen Shot 2023-07-15 at 8 01 34 PM" src="https://github.com/stemmlerjs/ddd-forum-bp-first/assets/6892666/a092342f-6b47-4d1e-9b8f-24884903c50d">

Reply:
@stemmlerjs,

> Is it right to call this a build tool?

Suggest your name. 🙂 How about just `cli`?

Reply:
@karpov-kir cli is perfect. I wrote some more notes on this and what I expect students to be able to do for the first E2E on the backend assignment. https://khalilstemmler.notion.site/Action-E2E-test-on-the-Backend-418f445837c4470b8bc5a1c1449960d4?pvs=4

Reply:
@stemmlerjs, done in https://github.com/stemmlerjs/ddd-forum-bp-first/pull/7/commits/60cd9664d7f1e1793a2d567ce6a68b51408c7248.